playbooks/files/plugins/cinder_api_local_check.py

```diff
@@ -53,11 +53,11 @@ def check(auth_ref, args):
     try:
         vol = s.get('%s/volumes/detail' % VOLUME_ENDPOINT,
                     verify=False,
-                    timeout=10)
+                    timeout=5)
```

Comment:
timeout changed from 10 to 5, when locally collecting metrics it can take several minutes per collection cycle when many systems are down. In a future PR I will multi-thread the telegraf plugin runner.